demo/app.component.ts

```diff
@@ -107,6 +108,25 @@ export class AppComponent implements OnInit {
     this.form2 = this.formBuilder.group({ 'range': [ [ 2, 8 ] ] });
   }
 
+  public onSliderInit(slider: any) {
+    this.sliderObject  = slider;
+    console.debug('Slider Object Instance Initiated and available for use in your component');
+    console.debug(this.sliderObject);
+  }
+
+  public resetSlider() {
+    this.sliderObject.reset();
+  }
+
+  public togglePips() {
```

Comment:
All of this goes right over my head. I started playing with angular just a couple of weeks ago :-). I just really need to be able dynamically change the maximum limit on the slider...